test/pytest/test_example_near_real_time_video.py

```diff
@@ -61,6 +61,7 @@ def create_mar_file(work_dir, session_mocker, model_archiver):
         ).as_posix(),
         export_path=work_dir,
         requirements_file=None,
+        config_file=None,
```

Comment:
cam we put this at the end of the arguments? If it's an optional value it might break code that relies on positional keywords